lib/features/email/presentation/widgets/calendar_event_information_widget.dart

```diff
@@ -0,0 +1,63 @@
+
+import 'package:core/presentation/extensions/color_extension.dart';
+import 'package:core/presentation/utils/responsive_utils.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:jmap_dart_client/jmap/mail/calendar/calendar_event.dart';
+import 'package:tmail_ui_user/features/email/presentation/styles/calendar_event_information_widget_styles.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/calendar_date_icon_widget.dart';
+
+class CalendarEventInformationWidget extends StatelessWidget {
+
+  final CalendarEvent calendarEvent;
+
+  const CalendarEventInformationWidget({
+    super.key,
+    required this.calendarEvent
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    final responsiveUtils = Get.find<ResponsiveUtils>();
```

Comment:
IMO, you should move out of build method

Reply:
It is a stateless widget, we can initialize the variables in the build function. If you initialize outside the stateless this widget will not have `const constructor`